docs/resources/work_in_progress.rb

```diff
@@ -0,0 +1,52 @@
+require 'chef/provisioning/azure_driver'
+with_driver 'azure'
+
+# NOTE: THESE ARE ASPIRATIONAL RESOURCES, SUBJECT TO CHANGE
+# THEY ALSO DO NOT STORE A MANAGEDENTRY IN CHEF SERVER
+# USE ONLY TO CREATE UNMANAGED RESOURCES IN AZURE
+#
+# USE AT OWN RISK!!
+
+## Auxiliary resources
+
+azure_storage_account 'spteststorage41' do
+  action :create
+  options :location => 'West US',
+          :geo_replication_enabled => false
+end
+
+azure_cloud_service 'sptestcloud41' do
+  action :create
+  options :location => 'West US'
+end
+
+## PAAS resources
+
+# note: azure_sql_server does not accept being provided a name until APIv2 (RM)
+# Azure will autogenerate a name, we should store this I guess so it can be used
+# later on...
+azure_sql_server 'sptestsql41' do
+  action :create
+  options :location => 'West US',
+          :login => 'sqluser',
+          :password => 'P2ssw0rd',
```

Comment:
To be clear, this is not an Azure password, this is the password for a newly-created login on a instance of SQL Azure created using credentials that are stored elsewhere and does need to be passed to the Azure SDK at converge time.

I generally follow the principle that example recipes should not rely on external dependencies such as attributes, or environment-specific solutions such as chef-vault and don't see much precedent set in the other drivers.  What's the feeling in general about this?  Should I update this example with a comment to explicitly call out that secrets need to be managed in a real environment?